### PR TITLE
Fix for #64

### PR DESCRIPTION
**Pull request by Shemnei**
Fixed extra empty line when an `IF` block at the start of a file was not evaluated.

Related bug: #64

### CODE REVIEW

**Comment by Shemnei** [comment]
bors try

File: crates/punktf-lib/src/template/resolve.rs [hunk 1]
```diff
@@ -107,6 +107,9 @@ pub struct Resolver<'a, PV, DV> {
 	/// Session where all errors/diagnostic which occur during the resolving
 	/// process are recorded to.
 	session: Session,
+
+	/// Number of current block being processed. This is used for internal purposes.
+	current_block: usize,
```
Comment (michidk) [inline crates/punktf-lib/src/template/resolve.rs:112]: explain those "internal purposes"

Comment (michidk) [reply crates/punktf-lib/src/template/resolve.rs:112]: or reference funktion where its used

**Review by michidk — APPROVED** [review]
One comment could be improved. Otherwise great work!

**Comment by Shemnei** [comment]
I thought i was done thats why i marked it as ready, but i noticed this behavior is not exclusive to  the file start. So i am still working on it and trying to fix that behavior. Sorry to have made you review it already :disappointed:.

**Comment by michidk** [comment]
no worries!On Sep 13, 2021 20:12, Jonas Grawe ***@***.***> wrote:

I thought i was done thats why i marked it as ready, but i noticed this behavior is not exclusive to  the file start. So i am still working on it and trying to fix that behavior. Sorry to have made you review it already 😞.

**Comment by Shemnei** [comment]
Thanks for the review :smile:.

bors r+

**Pull request merged**